app/assets/stylesheets/active_admin/components/_status_tags.scss

```diff
@@ -1,16 +1,15 @@
 .status_tag {
-  background: darken($secondary-color, 15%);
-  color: #fff;
-  text-transform: uppercase;
```

Comment:
Hmm maybe I'm just used to seeing it the other way. @timoschilling or anyone else want to chime in?